File: php/sort/tests/BubbleSort/BubbleSortTest.php
<?php

namespace Sort\Test\BubbleSort;

use PHPUnit\Framework\TestCase;
use Sort\BubbleSort\BubbleSort;

class BubbleSortTest extends TestCase
{
    public function testBubbleSort(): void
    {
        self::assertEquals([1, 3, 5, 7, 9], BubbleSort::sort([9, 3, 5, 1, 7]));
    }
}

File: java/sort/src/main/java/sort/bubble/BubbleSortTest.java
package sort.bubble;

import sort.util.SortUtil;

public class BubbleSortTest {

    public static void main(String[] args) {
        int[] a = {9, 7, 3, 1, 5};
        SortUtil.dump(BubbleSort.sort(a));

        int[] b = {10, 8, 2, 4, 6};
        SortUtil.dump(BubbleSort.sort(b));
    }

}
